src/systems/filecoin_markets/order/order.id

```diff
@@ -1,7 +1,10 @@
-type Ask struct {
-
-}
-
-type Bid struct {
-
+type StorageAsk struct {
+    Price         BigInt            // attoFIL per GiB per epoch
+    Collateral    BigInt            // TODO define units
+    
+    MinPieceSize  uint64
+    Miner         address.Address
+    Timestamp     uint64
```

Comment:
Should Timestamp be in terms of block.ChainEpoch? and Expiry/MaxDuration will be in block.ChainEpoch too.

Reply:
That sounds right, will add.